Tests-CI/tp.md

```diff
@@ -8,7 +8,7 @@ Tests are small pieces of code ensuring that a part of a program is working as e
 
 ### Why tests are useful
 
-It is from the utterly importance to implement tests **during** the development step. It will help you to ensure:
+It is why we place utter most importancing in implement tests **during** the development step. It will help you to ensure:
```

Comment:
It is why we place utter most importancing in implement
->
It is why we place utter most importance in implementing